core/server/api/v2/utils/serializers/output/utils/date.js

```diff
@@ -0,0 +1,10 @@
+const moment = require('moment-timezone');
+const settingsCache = require('../../../../../../services/settings/cache');
+
+const format = (date) => {
+    return moment(date)
+        .tz(settingsCache.get('active_timezone'))
+        .toISOString(true);
```

Comment:
These 3 lines above are everything that this PR is meant to do. The output is an ISO date with offset information - https://momentjs.com/docs/#/displaying/as-iso-string/